main/integration-tests/README.md

```diff
@@ -8,16 +8,20 @@
 
 ## Setup and Execution:
 
-   Create a Test Administrator user in your Service Workbench deployment where you want to run integration tests. 
-   Currently only internal auth provider can be used for authenticating this user.
-
-   Create/Locate the test config file in your local repository (placed in `main/integration-tests/config/settings/<STAGE>.yml`).
-   Enter the Test Administrator credentials and Test Project ID values. This file will be ignored by git when you make any changes. (Refer to the `main/integration-tests/config/settings/example.yml` file for guidance)
-   
-   Note: This file is unique from the rest of the `<STAGE>.yml` files created in the SDCs in that it does not gather serverless settings passed on from the hierarchies above (eg. from `main/condif/settings/.defaults.yml`)
-   
-   Now run the command below to trigger the integration test suite. 
-   Note that this will generate test-related resources in the deployment linked to the provided stage name.
+Create a Test Administrator user in your Service Workbench deployment where you want to run integration tests.
+Currently only internal auth provider can be used for authenticating this user.
+
+Create/Locate the test config file in your local repository (placed in `main/integration-tests/config/settings/<STAGE>.yml`).
+Enter the Test Administrator credentials and Test Project ID values. This file will be ignored by git when you make any changes. (Refer to the `main/integration-tests/config/settings/example.yml` file for guidance)
+
+Note: In the future, the password will no longer be stored in the yaml file, instead it will be stored in the parameter store.
+
+Note: This file is unique from the rest of the `<STAGE>.yml` files created in the SDCs in that it does not gather serverless settings passed on from the hierarchies above (eg. from `main/config/settings/.defaults.yml`)
+
+Now run the command below to trigger the integration test suite.
+Note that this will generate test-related resources in the deployment linked to the provided stage name.
+
+**NOTE** a PR will be made available in a few days so populating ENV_NAME and API_ENDPOINT is no longer needed.
```

Comment:
ENV_NAME (aka stage name) will still be needed, I assumed. Unless we were thinking of adding the `awsProfile` field in this file. Or is there another way?

Reply:
Correct, we need awsProfile and a few more properties to be added as we need to access AWS resources to do the cleanup logic and to access the cloudformation output